### ISSUE TITLE
[Design System Toolbox] Use the tag to show the Component Design Version

### ISSUE THREAD

**Issue by B3nz01d** [Orange-OpenSource/ouds-flutter#501]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-flutter/issues) for duplicate or closed feature requests

### Description

**As a user** of the demo app
**I want** to see the Component Design Version of each component
**So that** the version number uses the information tag component

AND
 **As a user** of the demo app
**I want** to see the about page
**So that** all the version numbers to use the information tag component

### Motivation and context

consistency with web documentation

the tag should be using the following tag setup:
* the information status
* rounded
* muted
* text only
* in small size

**Comment by B3nz01d** [comment]
- [ ] Android: [# 940](https://github.com/Orange-OpenSource/ouds-android/issues/940)
- [ ] iOS: [# 1176](https://github.com/Orange-OpenSource/ouds-ios/issues/1176)
- [ ] Flutter: [# 501](https://github.com/Orange-OpenSource/ouds-flutter/issues/501)

**Comment by B3nz01d** [comment]
For the About page:
- [ ] Android: [# 979](https://github.com/Orange-OpenSource/ouds-android/issues/979)
- [ ] iOS: [# 1203](https://github.com/Orange-OpenSource/ouds-ios/issues/1203)
- [ ] Flutter: [# 501](https://github.com/Orange-OpenSource/ouds-flutter/issues/501)

**Comment by nouha06** [comment]
📣 New Firebase App Distribution Alpha upload available 🚀 (Android)

Please, get the new build from App Tester app, or contact the project maintainers (@AhmedAmineZr ) to get access.

Display name: DesignToolbox (ALPHA) (609)
Version: 1.0.0
Build type: Alpha
Build details (GitHub): 551
cc @B3nz01d (product owner)
cc @AhmedAmineZr @nouha06 (dev team)
cc @jerome-regnier  (design team)

**Comment by nouha06** [comment]
New TestFlight Alpha upload available 🚀 (iOS)

Please, get the new build from TestFlight app, or contact the project maintainers (@AhmedAmineZr ) to get access.

Display name: Design Toolbox Flutter (609)
Version: 1.0.0
Build type: ALPHA
Build details (GitHub): 551
cc @B3nz01d (product owner)
cc @AhmedAmineZr @nouha06 (dev team)
cc @jerome-regnier  (design team)

**Comment by jerome-regnier** [comment]
👌 OK Design
